Glium 0.26.0-alpha3 does not compile with Glutin 0.22.0-alpha4/5

In a fresh package, if you just put `glium = "0.26.0-alpha3"` as dependency in your Cargo.toml, it will give you this error:
```
error[E0004]: non-exhaustive patterns: `Err(FunctionUnavailable)` not covered
   --> [...]/glium-0.26.0-alpha3/src/backend/glutin/mod.rs:253:15
    |
253 |         match self.borrow().swap_buffers() {
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ pattern `Err(FunctionUnavailable)` not covered
    |
```
There is a new enum variant since Glutin 0.22.0-alpha4, that is not considered in Glium.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "glium"
-version = "0.26.0-alpha3"
+version = "0.26.0-alpha5"
 authors = ["Pierre Krieger <pierre.krieger1708@gmail.com>"]
 description = """
 Elegant and safe OpenGL wrapper.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,7 +34,7 @@ unstable = [] # used for benchmarks
 test_headless = []  # used for testing headless display
 
 [dependencies.glutin]
-version = "0.22.0-alpha3"
+version = "=0.22.0-alpha5"
 features = []
 optional = true
 

--- a/src/backend/glutin/mod.rs
+++ b/src/backend/glutin/mod.rs
@@ -254,6 +254,11 @@ unsafe impl Backend for GlutinBackend {
             Ok(()) => Ok(()),
             Err(glutin::ContextError::IoError(e)) => panic!("I/O Error while swapping buffers: {:?}", e),
             Err(glutin::ContextError::OsError(e)) => panic!("OS Error while swapping buffers: {:?}", e),
+            // As of writing the FunctionUnavailable error is only thrown if
+            // you are swapping buffers with damage rectangles specified.
+            // Currently we don't support this so we just panic as this
+            // case should be unreachable.
+            Err(glutin::ContextError::FunctionUnavailable) => panic!("function unavailable error while swapping buffers"),
             Err(glutin::ContextError::ContextLost) => Err(SwapBuffersError::ContextLost),
         }
     }

